### PR TITLE
Added FxOS widget and button

### PR DESCRIPTION
**Pull request by decause**

### CODE REVIEW

File: hubs/static/js/fxos.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(function (){
+    var btn = document.getElementById("mozinstall");
+    console.log("mozinstall?")
+    btn.addEventListener("click", function(e) {
+        navigator.mozApps.install("http://127.0.0.1:8000/static/manifest.webapp");
```
Comment (ralphbean) [inline hubs/static/js/fxos.js]: We can't really hardcode this address here.  It needs to be somehow configurable.

File: hubs/static/manifest.webapp [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Fedora Hubs",
+  "description": "RIT Humanitarian Free/Open Source Software Development Course",
```
Comment (ralphbean) [inline hubs/static/manifest.webapp]: And this description needs to be updated.

File: hubs/templates/hubs.html [hunk 3]
```diff
@@ -92,4 +98,19 @@
     });
 });
 </script>
+<script>
+$(document).ready(function (){
+    $('#mozinstall').click(function() {
+        navigator.mozApps.install("http://127.0.0.1:8000/static/manifest.webapp");
```
Comment (ralphbean) [inline hubs/templates/hubs.html]: Same here, we can't hardcode this URL.  But, didn't this code appear somewhere else in the pull request also?

File: hubs/widgets/fxos.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import jinja2
+
+import hubs.validators as validators
+
+
+template = jinja2.Template("""
+    <script>
+    $(document).ready(function (){
+        $('#mozinstall').click(function() {
+            navigator.mozApps.install("http://127.0.0.1:8000/static/manifest.webapp");
```
Comment (ralphbean) [inline hubs/widgets/fxos.py]: And, it's hardcoded here too?  We can't have the code be duplicated in three places, I think.

File: hubs/widgets/fxos.py [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+            console.log("mozinstalled!")
+        });
+        /*
```
Comment (ralphbean) [inline hubs/widgets/fxos.py]: Also, this commented-out block can just be removed.  No need to keep it here.

**Comment by decause** [comment]
Nod nod nod. On it.

**Pull request closed without merging**